go/libraries/doltcore/merge/merge_schema.go

```diff
@@ -790,11 +784,11 @@ func mergeChecks(ourChks, theirChks, ancChks schema.CheckCollection) ([]schema.C
 			return nil, nil, err
 		}
 
-		// Check that all overlapping checks on columns are the same check
+		// TODO: redundant for checks that are defined exactly the same
 		for _, col := range colNames {
-			// Check if this column is referenced in their new checks
+			// See if this column is referenced in their new CHECKs
 			if _, ok := theirNewChkColsMap[col]; ok {
-				// If the two checks that reference this column are not the same, add conflict
+				// CONFLICT: our and their CHECK reference the same column and are not the same CHECK
```

Comment:
should this be a conflict? 
Both branches add checks to a column, but they have different names and definitions. Seems ok to me?